data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/Main.java

```diff
@@ -81,7 +82,9 @@ public static void start(
         Properties consumerConfig = Configurations.readPropertiesSync(env.getConsumerConfigFilePath());
         consumerConfig.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, KeyDeserializer.class.getName());
         consumerConfig.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, CloudEventDeserializer.class.getName());
-        consumerConfig.put(ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG, InvalidCloudEventInterceptor.class.getName());
+        consumerConfig.put(
+                ConsumerConfig.INTERCEPTOR_CLASSES_CONFIG,
+                InvalidCloudEventInterceptor.class.getName() + "," + NullCloudEventInterceptor.class.getName());
```

Comment:
Shouldn't the order be the opposite ?

Reply:
Maybe? I thought from our original conversation you said to to chain `InvalidCloudEventInterceptor` -> `NullCloudEventInterceptor`, maybe because the Invalid interceptor will still return ConsumerRecords with null data?

If I'm honest, I'm not super confident on which one should go first vs. second

Reply:
If we add tests we will know the correct order https://github.com/knative-extensions/eventing-kafka-broker/pull/3475#pullrequestreview-1764542407